### DISCUSSION TITLE
docker-compose安装v1.10.0后SQL查询PG数据库无法查看表结构

### DISCUSSION THREAD

**Discussion by yzb420** [hhyo/Archery#2288] [Q&A] [answered]
大佬好
     docker-compose安装v1.10.0后SQL查询，PG数据库无法查看表结构
这是v1.9.1的：
![image](https://github.com/hhyo/Archery/assets/102197195/f85d2fb0-ea3b-48f6-aa5e-8d6049b749a5)
这是v1.10.0的：
![image](https://github.com/hhyo/Archery/assets/102197195/35d7db78-044e-4928-a784-83d5c3f93cd4)

**Comment by LeoQuote** [comment, marked as answer]
在解决了. https://github.com/hhyo/Archery/pull/2283

**Comment by dongwanyang** [comment]
我安装的版本是1.10.0，oracle和sqlserver的表结构不能正常显示，只显示字段属性列，其他信息都没有，你们有遇到吗

**Reply by LeoQuote** [reply]
试试 master 版本 `ghcr.io/hhyo/archery:master`

**Reply by dongwanyang** [reply]
mater版本修复了吗

**Reply by LeoQuote** [reply]
哥, 你试试呗

**Reply by dongwanyang** [reply]
大佬，可以找到具体再那块修改代码吗，感谢感谢。

**Reply by dongwanyang** [reply]
我修改个别代码，怕拉取最新的代码，把我之前修改的都覆盖了

**Reply by dongwanyang** [reply]
大哥，我刚才试了你发的镜像，Orcle和SQLserver还是只返回字段属性列，其他列都不反回

**Reply by hhyo** [reply]
旧版本能正常展示吗

**Reply by yzb420** [reply]
旧版本是可以正常展示的
![image](https://github.com/hhyo/Archery/assets/102197195/0d9e0947-1625-4f0c-9589-070e86c799d0)
